js/ui/html_editor/utils/table_helper.js

```diff
@@ -0,0 +1,35 @@
+const TABLE_FORMATS = ['table', 'tableHeaderCell'];
+const TABLE_OPERATIONS = [
+    'insertTable',
+    'insertHeaderRow',
+    'insertRowAbove',
+    'insertRowBelow',
+    'insertColumnLeft',
+    'insertColumnRight',
+    'deleteColumn',
+    'deleteRow',
+    'deleteTable'
+];
+
+function getTableFormats(quill) {
+    const tableModule = quill.getModule('table');
+    return tableModule?.tableFormats ? tableModule.tableFormats() : TABLE_FORMATS; // backward compatibility with previous devextreme-quill versions
```

Comment:
seems like this row length is out of 80 characters. Mb we should move comment to the previous row?